lib/rubocop/result_cache.rb

```diff
@@ -91,6 +91,7 @@ def initialize(file, team, options, config_store, cache_root = nil)
       @allow_symlinks_in_cache_location =
         ResultCache.allow_symlinks_in_cache_location?(config_store)
       @path = File.join(cache_root,
+                        'rubocop_cache',
```

Comment:
When `ResultCache.cache_root(config_store)` (L90) is assigned instead of `options[:cache_root]` (L89) seems to create an unexpected nested `rubocop_cache/rubocop_cache` directory. Can you add a test to prevent the unexpected behavior and fix the implementation?

Reply:
Sorry, you're right. I've fixed the implementation.